app/views/api/v2/hosts/show.json.rabl

```diff
@@ -25,3 +25,16 @@ end
 child :config_groups do
   extends "api/v2/config_groups/main"
 end
+
+host_additional_views(@host).each do |id, tab|
+  if tab.is_a? String
+    node do |host|
+      partial "api/v2/#{tab}", :object => host
```

Comment:
I see that the "reuse the same structure in API and UI" approach doesn't play well... I think I'll just create two extra extension points to the configuration: `#api_host_view` and `#api_host_list`